SimCalorimetry/HcalSimProducers/python/hcalUnsuppressedDigis_cfi.py

```diff
@@ -50,6 +52,9 @@
 from Configuration.StandardSequences.Eras import eras
 if eras.fastSim.isChosen():
     hcalSimBlock.hitsProducer = cms.string('famosSimHits')
+if eras.run2_HF_2016.isChosen():
```

Comment:
@makortel @Dr15Jones - Thanks for the info about `toModify()`. I'll correct the usage in this PR (and also fix the FastSim one while I'm at it, attn @lveldere).